attributes/default.rb

```diff
@@ -1 +1,25 @@
+# 'initial_url_required' is there to preserve backwards compatable
+# behavior with the original Rackspace version
+# Override it to false to not require 'initial_url' in your
+# environment, role or wrapper cookbook
+default['scaleft']['initial_url_required'] = true
```

Comment:
I would like to see this removed instead in favor of logic that checks if `['scaleft']['auto_enroll']` is true, then either `['scaleft']['initial_url']` or `['scaleft']['token_file']` need to be set.

Honestly I think we only had this check in place because Rackspace utilizes the SaaS version of ScaleFT so we want'ed to catch and enforce using the `initial_url`. I like making this less opinionated and match the logic of the config file if you are doing this.